pygsti/circuits/circuit.py

```diff
@@ -1535,13 +1602,12 @@ def insert_idling_lines_inplace(self, insert_before, line_labels):
         -------
         None
         """
-        #assert(not self._static),"Cannot edit a read-only circuit!"
-        # Actually, this is OK even for static circuits because it won't affect the hashed value (labels only)
+        assert(not self._static),"Cannot edit a read-only circuit!"
         if insert_before is None:
-            i = len(self.line_labels)
+            i = len(self._line_labels)
         else:
-            i = self.line_labels.index(insert_before)
-        self._line_labels = self.line_labels[0:i] + tuple(line_labels) + self.line_labels[i:]
+            i = self._line_labels.index(insert_before)
+        self.line_labels = self._line_labels[0:i] + tuple(line_labels) + self._line_labels[i:]
```

Comment:
Why did the LHS get moved to the property? Can we not just assign to `_line_labels` still?